app/views/effective/events_mailer/event_registrant_confirmation.html.haml

```diff
@@ -1,22 +1,39 @@
-%h2 Event Registrant Confirmation
+%h1{style: "text-align: center"} Your tickets have been confirmed!
 
-%p Event: #{@event.title}
-%p Date: #{effective_events_event_schedule(@event)}
-%p Ticket: #{@event_registrant.purchasable_name.gsub('<br>', ' - ').html_safe} - #{price_to_currency(@event_registrant.price)}
-
-- if @event_registrant.responses.present?
-  %p= @event_registrant.responses
-
-- if @event_registration.present? && @event_registrant.user == @event_registration.owner
-  %p= link_to('View your registration', effective_events.event_event_registration_url(@event, @event_registration))
+%p{style: "text-align: center"}
+  %strong{style: "font-size: 18px"}= @event.title
+  %br
+  #{effective_events_event_schedule(@event)}
 
 - if @event.rich_text_confirmation_email.present?
   %hr
   = @event.rich_text_confirmation_email.to_s.html_safe
 
-- if @event.delayed_payment?
-  %strong Please note:
-  %ul
-    %li Credit card payment will be processed on #{@event.delayed_payment_date.strftime("%A, %B %d, %Y")} 
+- if @event.delayed_payment? && @event_registration.present? && @event_registrant.user == @event_registration.owner
```

Comment:
i only want to show this message if they actually have a pending payment, is this the correct way to do it?